test-data/unit/check-incomplete-fixture.test

```diff
@@ -16,9 +16,7 @@ m.x # E: "object" has no attribute "x"
 from typing import Dict
 def f(x: Dict[int]) -> None: pass
 [out]
-main:1: error: Module "typing" has no attribute "Dict"
```

Comment:
could you update this test with something more exotic from typing.pyi?

Reply:
I can try, but if having dict and list is an invariant now, maybe this test should just go away?

Reply:
There are other things in typing.pyi that probably shouldn't be in the default fixture, e.g. like dataclass_transform. This error is still useful there.

(In general, if we can get away with omitting something from the fixture and having devs hit this error, I think that's much more preferable than having an empty class, where degradation from missing API is less obvious. I'm fine with this PR for list and dict, because they're fundamental enough that their complete absence is annoying)

Reply:
This leads me to think that maybe the "Maybe your test fixture does not define ..." message should be triggered by attribute access to a type that was defined in a fixture, not only absence of the type.

Reply:
In its current state, this test can be removed, and "builtins.list" and "builtins.dict" can be removed from `SUGGESTED_TEST_FIXTURES`, since we'd no longer have a "trip wire" for devs.

I do think this trip wire can remain useful if we warn about type missing _or_ type's attributes missing, but
a) I can't tell how noisy that'd be
b) Implementing something like this would require determining whether a TypeInfo comes a fixture (or, perhaps, if its `defn` comes from a fixture). I can't see anything on either of them that allows me to know it right now. Before investing more work, I'm curious what people think.

Reply:
Thanks for updating! And sorry my message was confusing, didn't realise there were several other testXMissingFromStubs tests.

I think logic for b) we could use is just checking if module is typing.pyi or builtins.pyi?